docs/en/operations/startup-scripts.md

```diff
@@ -17,6 +17,10 @@ ClickHouse can run arbitrary SQL queries from the server configuration during st
             <query>CREATE TABLE TestTable (id UInt64) ENGINE=TinyLog</query>
             <condition>SELECT 1;</condition>
         </scripts>
+        <scripts>
+            <query>CREATE DICTIONARY test_dict (...) SOURCE(CLICKHOUSE(...))</query>
```

Comment:
We should've also tested the case when the user is not `default`, but has the permission to create dictionaries.